sample/index.html

```diff
@@ -35,6 +36,8 @@ <h2>CKEditor 5 online builder demo - DecoupledDocumentEditor build</h2>
 			</div>
 			<div class="row row-editor">
 				<div class="editor">
+					<span class="mention">@swarley</span>
+					<span class="mention">@lilypad</span>
```

Comment:
Added these as examples of existing mentions that will be updated when CKEditor opens